VideoLocker/res/values/strings.xml

```diff
@@ -261,11 +261,17 @@
     <!-- The label to use when indicating a post was authored by a privileged user -->
     <string name="discussion_priviledged_author_attribution">By %s</string>
     <!-- The date and author of a discussion comment, question, or thread, e.g. "3 days ago by Brian STAFF" -->
```

Comment:
By the way, these examples should also be updated to reflect the formatting changes to the author label in #646 and #646, i.e small case and with parenthesis.